lib/cdo/shared_constants/pd/principal_approval_application_constants.rb

```diff
@@ -30,9 +30,6 @@ def self.clean_multiline(string)
       pacific_islander: 'Native Hawaiian or other Pacific Islander',
       american_indian: 'American Indian or Native Alaskan',
       other: 'Other',
-      replace_course: 'Will this course replace an existing computer science course in the master schedule?',
-      understand_fee: 'By checking this box, you indicate that you understand there may be a fee for the professional learning program your teacher attends.',
-      pay_fee: 'If there is a fee for the program, would your school be able to pay for the fee?',
       contact_invoicing: "Contact name for invoicing (if applicable)",
       contact_invoicing_detail: "Contact email or phone number for invoicing (if applicable)",
```

Comment:
If the contact invoicing questions are being removed, we can remove these too I think!